ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -1605,4 +1604,20 @@ FCU_DFSS_DFVSC_FDPM_CHWDC_SRC_DFRMM:
   - FDPM
   - CHWDC
   - SRC
-  - DFRMM 
\ No newline at end of file
+  - DFRMM
+
+FCU_RHDHS_DTM_DFSS_STM_DTC_SHM_SFVSC_ALM_SS_SRC:
+  description: "FCU with temperature control, humidity monitor with electrical parameter monitoring."
+  is_canonical: true
+  implements:
+  - FCU
+  - RHDHS
+  - DTM
+  - DFSS
+  - STM
+  - DTC
+  - SHM
+  - SFVSC
+  - ALM
+  - SS
```

Comment:
is this in addition to the fan commands?

Reply:
I changed this now to:

```
FCU_RHDHS_DXSC_SHM_RTC:
  description: "FCU with temperature control, humidity monitor with electrical parameter monitoring and water leak alarm."
  is_canonical: true
  implements:
  - FCU
  - RHDHS
  - DXSC
  - SHM
  - RTC
  uses:
  - water_leak_detection_alarm
  - supply_fan_run_status
  - low_discharge_air_flowrate_alarm
  - failed_alarm
```

alarms are now included here, rather than in a tag